src/scrapers/poalim-business/helpers.js

```diff
@@ -0,0 +1,312 @@
+import moment from 'moment';
+import fs from 'fs';
+import {
+  getRestContext,
+  DATE_FORMAT,
+  fetchPoalimXSRFWithinPage,
+  fetchPoalimXSRFWithinPageGet,
+  convertTransactions,
+} from '../hapoalim';
+import { fetchGetWithinPage } from '../../helpers/fetch';
+import { addMonths } from '../../helpers/dates';
+
+export async function fetchAccountData(page, options) {
```

Comment:
very similar to `fetchAccountData` from Poalim and I'm sure we can share a lot.
But I need someone to work with on that or to test those things.

I take out much more information but I think it might be relevant also to Poalim, just need to make sure it works